### PR TITLE
V3 add DocumentVerificationReason java sdk. DTONBWTWO-132

### PR DESCRIPTION
**Pull request by simrandsingh**
add rejection reasons java sdk. version 3

### CODE REVIEW

**Review by arao6 — COMMENTED** [review]
Please see the comments on PR #124

File: src/main/java/com/hyperwallet/clientsdk/model/DocumentVerificationReason.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.hyperwallet.clientsdk.model;
+
+public enum DocumentVerificationReason {
+    DOCUMENT_EXPIRED("Document in expired"),
+    DOCUMENT_NOT_RELATED_TO_PROFILE("Document does not match account information"),
```
Comment (hrandhawa-hw) [inline src/main/java/com/hyperwallet/clientsdk/model/DocumentVerificationReason.java:5]: These descriptions need to be removed here too.

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVerificationDocument.java [hunk 2]
```diff
@@ -22,6 +23,8 @@
 
     private String country;
 
+    private List<HyperwalletDocumentRejectReason> reasons;
+
```
Comment (hrandhawa-hw) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVerificationDocument.java:27]: Needs createdOn too

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java [hunk 3]
```diff
@@ -63,6 +64,7 @@
     private String country;
     private String language;
     private String programToken;
+    private List<HyperwalletVerificationDocument> documents;
```
Comment (hrandhawa-hw) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java:67]: Wow, V3 User never had documents?

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 4]
```diff
@@ -7562,4 +7563,64 @@ public void testUploadUserDocuments_uploadFilesMissing() {
...
+        HyperwalletUser hyperwalletUser = getHyperwalletUser();
+        final String url = "https://api.sandbox.hyperwallet.com/rest/v3/users/" + "test-userToken";
+        Mockito.when(mockApiClient.get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class))).thenReturn(hyperwalletUser);
+        HyperwalletUser user = client.getUser("test-userToken");
+        Mockito.verify(mockApiClient).get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class));
+        assertThat(user.getToken(), is(hyperwalletUser.getToken()));
+        assertThat(user.getDocuments().size(), is(1));
+        assertThat(user.getDocuments().get(0).getStatus(), is("INVALID"));
+        assertThat(user.getDocuments().get(0).getReasons().size(), is(1));
+        assertThat(user.getDocuments().get(0).getReasons().get(0).getName(), is(DocumentVerificationReason.DOCUMENT_EXPIRED));
```
Comment (hrandhawa-hw) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java:7604]: needs createdOn assertion

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVerificationDocument.java (hunk 2 above)
Comment (simrandsingh) [reply src/main/java/com/hyperwallet/clientsdk/model/HyperwalletVerificationDocument.java:27]: added.

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java (hunk 3 above)
Comment (simrandsingh) [reply src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java:67]: yes, probably not used much by clients.

**Review by kverma-hw — APPROVED** [review]

**Review by arao6 — APPROVED** [review]

**Review by hrandhawa-hw — APPROVED** [review]

**Pull request merged**